src/core/server/saved_objects/routes/bulk_create.ts

```diff
@@ -40,6 +40,7 @@ export const registerBulkCreateRoute = (router: IRouter, { coreUsageData }: Rout
                 })
               )
             ),
+            originId: schema.maybe(schema.string()), // TODO: enable only in dev mode? we don't want external consumers to use this
```

Comment:
I don't like this, but atm we don't have the ability to set an object's `originId` and `initialNamespaces` at the same time using an external client.

Another idea is to use the Spaces Test Plugin to set up an HTTP route, only for this test suite, that exposes an unrestricted API for bulk-creating saved objects. That might be the best approach since this is all really just a stop-gap until imports are space-aware. WDYT?
https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/x-pack/test/spaces_api_integration/common/fixtures/spaces_test_plugin/server/plugin.ts#L13